backend/src/auth/auth.controller.ts

```diff
@@ -44,9 +45,16 @@ export class AuthController {
     description:
       API_RESPONSES_DESCRIPTION.SUCCESSFUL_SIGNUP_EMAIL_SENT_DESCRIPTION,
   })
+  @ApiCreatedResponse({
+    description:
+      API_RESPONSES_DESCRIPTION.SUCCESSFUL_SIGNUP_EMAIL_SENT_DESCRIPTION,
+  })
```

Comment:
does the `/local/signup` route return code 201? if it does then should remove the `@ApiOkResponse`, otherwise if success returns code 200 then should remove this

Reply:
Yeap, it returns code 201. Discovered that in the Jest testing too!